File: Android/01 - IFPB/app/src/main/java/br/com/valeriacavalcanti/ifpb/MainActivity.java
package br.com.valeriacavalcanti.ifpb;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    private TextView tvMensagem;
    private Button btnOk;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        this.tvMensagem = (TextView)findViewById(R.id.tvMainMensagem);
        this.btnOk = (Button)findViewById(R.id.btnMainOk);

//        this.btnOk.setOnClickListener(new View.OnClickListener() {
//            @Override
//            public void onClick(View view) {
//                ///MainActivity.this.tvMensagem.setText("IFPB");
//                tvMensagem.setText("IFPB");
//            }
//        });

        this.btnOk.setOnClickListener(new OnClickBotao());
    }

    private class OnClickBotao implements View.OnClickListener{
        @Override
        public void onClick(View view) {
            ///MainActivity.this.tvMensagem.setText("IFPB");
            tvMensagem.setText("IFPB");
        }
    }
}

File: Kotlin/05 - IFPBK/app/src/main/java/br/com/valeriacavalcanti/ifpbk/MainActivity.kt
package br.com.valeriacavalcanti.ifpbk

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    lateinit var tvMensagem: TextView
    lateinit var btnOk: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        this.tvMensagem = findViewById(R.id.tvMainMensagem) as TextView
        this.btnOk = findViewById(R.id.btnMainOk) as Button

//        this.btnOk.setOnClickListener(View.OnClickListener {
//            this.tvMensagem.setText("IFPB")
//        })

        this.btnOk.setOnClickListener({ atualiza(it) })
    }

    fun atualiza(view: View){
        this.tvMensagem.setText("IFPB")
    }
}
